WebDriverAgentLib/Categories/XCUIElement+FBScrolling.h

```diff
@@ -34,6 +34,15 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)fb_scrollRight;
 
 /**
+ Scrolls receiver in given direction
+ 
+ @param direction directon in which reciever will be scrolled.
+ @param error If there is an error, upon return contains an NSError object that describes the problem.
+ @return YES if the operation succeeds, otherwise NO.
+ */
+- (BOOL)fb_scrollInDirection:(NSString *)direction error:(NSError **)error;
```

Comment:
Adding it to `XCUIElement+FBScrolling.h` doesn't make much sense specially that we have `scrollDown` etc methods. I would keep it in `FBElementCommands.m`.